testing/trino-testing/src/main/java/io/trino/testing/LocalSpoolingManager.java

```diff
@@ -89,14 +96,19 @@ public SpooledSegmentHandle handle(SpooledLocation location)
         if (!(location instanceof CoordinatorLocation coordinatorLocation)) {
             throw new IllegalArgumentException("Cannot convert direct location to handle");
         }
-        return new LocalSpooledSegmentHandle(Paths.get(coordinatorLocation.identifier().toStringUtf8()));
+        try {
+            return HANDLE_CODEC.fromJson(coordinatorLocation.identifier().toStringUtf8());
```

Comment:
something is fishy here. `LocalSpooledSegmentHandle` does not seem like something to be referred as identifier in its serialized form.

Reply:
It's a shortcut that doesn't matter since this is a test only